libs/ledger/src/chain/block_coordinator.cpp

```diff
@@ -1142,10 +1194,26 @@ BlockCoordinator::State BlockCoordinator::OnReset()
 
   if (block != nullptr && !block->hash.empty())
   {
-    block_hash_->set(*reinterpret_cast<uint64_t const *>(block->hash.pointer()));
+    if ((block->block_number % 100) == 0)
```

Comment:
This is not going to make the block hash less noisy but whatever not really an issue